kanidmd/Dockerfile

```diff
@@ -1,22 +1,21 @@
 FROM opensuse/tumbleweed:latest
 MAINTAINER william@blackhats.net.au
 
-EXPOSE 8080
-
 COPY . /home/kanidm/
 
 WORKDIR /home/kanidm/
 
 RUN zypper install -y timezone cargo rust gcc sqlite3-devel libopenssl-devel && \
-    RUSTC_BOOTSTRAP=1 cargo build --release && \
-    zypper rm -u -y cargo rust gcc && \
-    zypper clean
+    RUSTC_BOOTSTRAP=1 cargo build --release
 
-RUN cd /etc && \
-    ln -sf ../usr/share/zoneinfo/Australia/Brisbane localtime
+FROM opensuse/tumbleweed:latest
+
+EXPOSE 8080
```

Comment:
might as well be consistent between the getting started docs and the dockerfile?

Reply:
I don't get this comment?